service/src/main/java/uk/gov/hmcts/reform/fpl/service/CaseService.java

```diff
@@ -15,16 +16,23 @@
 
     private final CaseUserApi caseUserApi;
     private final AuthTokenGenerator authTokenGenerator;
+    private final RequestData requestData;
 
     @Autowired
-    public CaseService(CaseUserApi caseUserApi, AuthTokenGenerator authTokenGenerator) {
+    public CaseService(CaseUserApi caseUserApi, AuthTokenGenerator authTokenGenerator, RequestData requestData) {
```

Comment:
could we use `@RequiredArgsConstructor(onConstructor = @__(@Autowired))` in place of the constructor? I think it's neater 😃